<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsView</class>
 <widget class="QWidget" name="CSettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidgetSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Video</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Artificial Intelligence</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mod Repositories</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="settingsScrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>100</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>602</width>
        <height>848</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="2,0,1,1,1">
       <item row="1" column="0">
        <widget class="QLabel" name="labelLanguage">
         <property name="text">
          <string>VCMI Language</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="labelReservedArea">
         <property name="text">
          <string>Reserved screen area</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="labelFramerateLimit">
         <property name="text">
          <string>Framerate Limit</string>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="labelCursorType">
         <property name="text">
          <string>Cursor</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="labelInterfaceScaling">
         <property name="text">
          <string>Interface Scaling</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="4">
        <widget class="QSpinBox" name="spinBoxNetworkPort">
         <property name="minimum">
          <number>1024</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>3030</number>
         </property>
        </widget>
       </item>
       <item row="14" column="1" colspan="4">
        <widget class="QSpinBox" name="spinBoxReservedArea">
         <property name="suffix">
          <string notr="true">%</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelNetworkPort">
         <property name="text">
          <string>Network port</string>
         </property>
        </widget>
       </item>
       <item row="30" column="0">
        <widget class="QLabel" name="labelRepositoryDefault">
         <property name="text">
          <string>Default repository</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="labelAutosaveCountLimit">
         <property name="text">
          <string>Autosave Count Limit</string>
         </property>
        </widget>
       </item>
       <item row="24" column="0">
        <widget class="QLabel" name="labelAlliedPlayerAI">
         <property name="text">
          <string>Adventure Map Allies</string>
         </property>
        </widget>
       </item>
       <item row="23" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxEnemyPlayerAI">
         <property name="currentText">
          <string notr="true">VCAI</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">VCAI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Nullkiller</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="25" column="0">
        <widget class="QLabel" name="labelNeutralAI">
         <property name="text">
          <string>Neutral AI in battles</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelAutosaveFrequency">
         <property name="text">
          <string>Autosave Frequency</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxResolution"/>
       </item>
       <item row="8" column="1" colspan="4">
        <widget class="QSpinBox" name="spinBoxAutosaveFrequency">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The interval (in-game days) between autosaves.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QCheckBox" name="checkBoxVSync">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="pushButtonTranslation">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="QLabel" name="labelTranslationStatus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="21" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxCursorType">
         <item>
          <property name="text">
           <string>Hardware</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Software</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="30" column="2" colspan="3">
        <widget class="QLineEdit" name="lineEditRepositoryDefault">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelAutosaveMode">
         <property name="text">
          <string>Autosave Mode</string>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QLabel" name="labelEnemyPlayerAI">
         <property name="text">
          <string>Adventure Map Enemies</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelTranslation">
         <property name="text">
          <string>Heroes III Translation</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="labelVideo">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Video</string>
         </property>
        </widget>
       </item>
       <item row="27" column="0">
        <widget class="QLabel" name="labelEnemyAI">
         <property name="text">
          <string>Enemy AI in battles</string>
         </property>
        </widget>
       </item>
       <item row="24" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxAlliedPlayerAI">
         <property name="currentText">
          <string notr="true">VCAI</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">VCAI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Nullkiller</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="31" column="2" colspan="3">
        <widget class="QLineEdit" name="lineEditRepositoryExtra">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="31" column="0">
        <widget class="QLabel" name="labelRepositoryExtra">
         <property name="text">
          <string>Additional repository</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1" colspan="4">
        <widget class="QSpinBox" name="spinBoxFramerateLimit">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="19" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxShowIntro">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>On</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="labelFullScreen">
         <property name="text">
          <string>Fullscreen</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="4">
        <widget class="QSpinBox" name="spinBoxAutosaveCountLimit">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Applicable solely in the context of Autosave Mode set to &amp;quot;Using Counter&amp;quot;.&lt;/p&gt;&lt;p&gt;This option defines the maximum number of autosaves to retain. Once the specified count is reached, new autosaves will be named starting from 0 (effectively overwriting the oldest saves).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="26" column="0">
        <widget class="QLabel" name="labelFriendlyAI">
         <property name="text">
          <string>Friendly AI in battles</string>
         </property>
        </widget>
       </item>
       <item row="28" column="0">
        <widget class="QLabel" name="labelRepositories">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Mod Repositories</string>
         </property>
        </widget>
       </item>
       <item row="26" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxFriendlyAI">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string notr="true">BattleAI</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">BattleAI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">StupidAI</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="15" column="1" colspan="4">
        <widget class="QSpinBox" name="spinBoxInterfaceScaling">
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="27" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxEnemyAI">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string notr="true">BattleAI</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">BattleAI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">StupidAI</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelLanguageBase">
         <property name="text">
          <string>Heroes III Data Language</string>
         </property>
        </widget>
       </item>
       <item row="29" column="3" colspan="2">
        <widget class="QPushButton" name="refreshRepositoriesButton">
         <property name="text">
          <string>Refresh now</string>
         </property>
        </widget>
       </item>
       <item row="29" column="0">
        <widget class="QLabel" name="labelAutoCheck">
         <property name="text">
          <string>Check on startup</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="labelVSync">
         <property name="text">
          <string>VSync</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelGeneral">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>General</string>
         </property>
        </widget>
       </item>
       <item row="31" column="1">
        <widget class="QCheckBox" name="checkBoxRepositoryExtra">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="labelDisplayIndex">
         <property name="text">
          <string>Display index</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxAutosaveMode">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Off&lt;/span&gt; - No autosaves&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Using Counter&lt;/span&gt; - Autosaves will be named incrementally unsing Autosave# format. Once Autosave Count Limit is reached, counter will start again from 0 overwriting existing (oldest) autosaves.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Using Turn Info&lt;/span&gt; - Autosaves will be named using Turn info. EG: Autosave112. Autosave Count limit will be disabled when this option is used.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Using Schema&lt;/span&gt; - Autosaves will follow a predefined schema.EG: %mapName%/%timestamp%/%turn%. For more details check documentation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Using Counter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Using Turn Info</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Using Schema</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="13" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxFullScreen">
         <property name="toolTip">
          <string>
											Select display mode for game

											Windowed - game will run inside a window that covers part of your screen

											Borderless Windowed Mode - game will run in a window that covers entirely of your screen, using same resolution as your screen.

											Fullscreen Exclusive Mode - game will cover entirety of your screen and will use selected resolution.
										</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Windowed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Borderless fullscreen</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Exclusive fullscreen</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="labelResolution">
         <property name="text">
          <string>Resolution</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxLanguage"/>
       </item>
       <item row="30" column="1">
        <widget class="QCheckBox" name="checkBoxRepositoryDefault">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="22" column="0">
        <widget class="QLabel" name="labelArtificialIntelligence">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Artificial Intelligence</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="labelShowIntro">
         <property name="text">
          <string>Show intro</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxLanguageBase"/>
       </item>
       <item row="25" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxNeutralAI">
         <property name="currentText">
          <string notr="true">BattleAI</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">BattleAI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">StupidAI</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="29" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxAutoCheck">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>On</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="20" column="1" colspan="4">
        <widget class="QComboBox" name="comboBoxDisplayIndex"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="labelAutosaveBeforeBattles">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Autosave Before Battles</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="checkBoxAutosaveBeforeBattles">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
